src/app/backing/components/DecorativeSquares/DecorativeSquares.tsx

```diff
@@ -18,15 +18,13 @@ export function DecorativeSquares({
       xmlns="http://www.w3.org/2000/svg"
       {...props}
     >
-      <rect width="50" height="40" fill="#1E1E1E" />
+      <rect width="50" height="40" />
```

Comment:
There is one square that does not change color.
<img width="109" height="55" alt="Screenshot 2025-07-20 at 7 13 11 PM" src="https://github.com/user-attachments/assets/4838a37a-548c-4efc-953a-6f1dd4653d61" />